### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by TekyRey**
### **Navigation**

In this project we:

- Added a `navbar`
- Added a contact form
- Linked `nav-links` to redirect to different sections of the web page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,8 +78,36 @@ <h1 class="pageTitle text-center">All Books</h1>
...
+            <div class="footer">
+              <p>
+                copy right    ...
+              </p>
+            </div>
+          </div>
+      </section>
         </div>
-        
+
```
Comment (ahmetbozaci) [inline index.html:106]: - [x] Kindly delete commented code to make your code clean.

- [x] I think this js file for adding book. That's why ı cant adds a book and if I open this comment navigation is not working. This is optional but to make the project well please fix it. ### OPTIONAL

image

File: scripts/class.js [hunk 2]
```diff
@@ -92,4 +92,59 @@ document.querySelector('#book-list').addEventListener('click', (e) => {
 
   Book.removeBook(e.target.parentElement.previousElementSibling.textContent);
 });
+
+const d = new Date();
+// const n = d.getFullYear();
```
Comment (ahmetbozaci) [inline scripts/class.js]: - [x] Kindly delete commented code to make your code clean.

File: style.css [hunk 3]
```diff
@@ -29,3 +29,36 @@ body {
...
+.title {
+  padding-top: 30px;
+  text-align: center;
+  font-size: 23px;
+  font-weight: 800;
+  padding-bottom: 10px;
+}
+
+.info {
+  display: flex;
```
Comment (ahmetbozaci) [inline style.css]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: style.css [hunk 4]
```diff
@@ -29,3 +29,36 @@ body {
...
+.footer {
+  border: 2px solid rgb(5, 5, 5);
+  background-image: url("back.jpg");
+  margin-bottom: 20px;
+  margin-top: 20px;
+  padding-top: 7px;
+  padding-left: 15px;
+  margin-left: 15px;
+  margin-right: 15px;
+}
```
Comment (ahmetbozaci) [inline style.css]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @TekyRey  @henrykgggg ✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- You added a navigation bar and it is working well.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Your commit messages should always start with capital letters and they should be an imperative tense. I see some of them are not like that. Instead of writing `Added style or Created the file` you should write `Add style or Create the file`
- ***I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this in your next projects.***

**OPTIONAL**

- You can change them by using `git-rebase`. If you want to know how to change ı leave a few resources.
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: scripts/class.js [hunk 5]
```diff
@@ -75,7 +71,6 @@ function display() {
             `;
   });
 }
-// add event listener to the form
 bookForm.addEventListener('submit', (e) => {
   e.preventDefault();
   const newBook = new Book(bookTitle.value, bookAuthor.value);
```
Comment (thecodechaser) [inline scripts/class.js:76]: - [ ] Adding books to the collection is not working, Book is also not being stored in local storage. Please fix this issue and make sure the users are able to add and delete books from the collection.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @TekyRey, and @henrykgggg 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -9,11 +9,37 @@
...
+      
+      Awesome Books
+    </a>
+
+    <div class ="">
+     <ul class="nav">
+  <li class="nav-item">
+    <a class="nav-link active" aria-current="page" href="#" id="list">List</a>
+  </li>
+  <li class="nav-item">
```
Comment (koachcode) [inline index.html:27]: - Good job so far trying to implement the localstorage, but its implementation does not save more than one book, it keeps changing what was previously in the storage and on refresh all list on the webpage disappears, Please try to fix it.

You can reach out to me on slack via `@kanu mike chibundu`

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @TekyRey , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

keep-trying-youre-almost-there

 ## Highlights
- No linter errors :+1: 
- Goos commit messages :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>

**Hello Team**, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**